packages/components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import type { StoryObj } from "@storybook/react";
+import React from "react";
+import Checkbox, { CheckboxInput, Label } from "./Checkbox";
+
+const defaultArgs = {
+  disabled: false,
+  label: "Checkbox",
+};
+
+export default {
+  title: "Checkbox",
+  component: Checkbox,
+  args: defaultArgs,
+  argTypes: {
+    // For some reason Storybook does not infer all props correctly;
+    // we manually include the most relevant controls here.
+    checked: {
+      control: "radio",
+      options: [true, false, "indeterminate"],
+    },
+    size: {
+      description: "Size of the checkbox label.",
+      control: "radio",
+      options: ["small", "medium"],
+      table: { defaultValue: { summary: "medium" } },
+    },
+  },
+};
+
+type Args = React.ComponentProps<typeof Checkbox>;
+
+/**
+ * Controlled example to make checked stories interactive.
+ */
+function CheckedExample(args: Args) {
+  const [checked, setChecked] = React.useState(true);
+  const handleChange = () => {
+    setChecked(!checked);
+  };
+
+  return <Checkbox checked={checked} onChange={handleChange} {...args} />;
+}
+
+export const Default: StoryObj<Args> = {};
+
+export const Checked: StoryObj<Args> = {
```

Comment:
went back to the old (in traject) structure, confirmed Docs page & Controls panel work:
![Kapture 2022-01-18 at 10 56 10](https://user-images.githubusercontent.com/15840841/150001186-07058809-a193-4abb-866c-3acd020dcef7.gif)